apps/web/src/app/pages/trade/bridge/types.ts

```diff
@@ -1,10 +1,15 @@
-import { Token } from '@balancednetwork/sdk-core';
+import { Currency, CurrencyAmount, Token } from '@balancednetwork/sdk-core';
 import { Event } from '@cosmjs/cosmwasm-stargate';
+import BigNumber from 'bignumber.js';
 
 export enum XWalletType {
```

Comment:
@hetfly This type is not created for the purpose? revert it.

Reply:
It's used for combined total of multiple tokens(ie. base ETH, arb ETH and bsc ETH) and for token value in dollars, can't use CurrencyAmount for it